packages/sanity/src/conditional-property/useConditionalProperty.test.tsx

```diff
@@ -1,32 +1,10 @@
 import {renderHook} from '@testing-library/react-hooks'
```

Comment:
we might be able to remove these tests - they should be obsolete after resolving conditional properties as part of preparing the form state now. Made a note to check if this is the case and can do that separately.

Reply:
Awesome, thanks for checking!